### PR TITLE
Added new field Instagram ID

### PR DESCRIPTION
**Pull request by TD-17**
issue #41

### CODE REVIEW

File: app/controllers/signup.js [hunk 1]
```diff
@@ -113,6 +115,14 @@ export default class SignupController extends Controller {
       required: true,
       showError: false,
     },
+    {
+      id: 'instagram_id',
+      label: 'Instagram ID (Not the full URL)',
+      type: 'text',
+      errorMessage: 'Instagram username is required',
+      required: true,
+      showError: false,
```
Comment (swarajpure) [inline app/controllers/signup.js:124]: This field would be optional. So we need not have `required: true`, please make it false.
errorMessage would be empty string in this case.

Also, can we please remove the `(Not the full URL)`? 
That was required for linkedin and twitter as many people might get confused and enter the full url (for example, `https://www.linkedin.com/in/swarajrajpure`)

When we talk about the instagram id, most people anyway use it from their phones and they very well know what's an `instagram id`

**Review by swarajpure — CHANGES_REQUESTED** [review]
Welcome to a project which uses ember! 🎉 

Requesting some minor changes, post which we can merge this.

Thank you! 😄

File: app/controllers/signup.js (hunk 1 above)
Comment (TD-17) [reply app/controllers/signup.js:124]: okay ! So I have to commit three changes ! On it !

**Review by swarajpure — APPROVED** [review]
Looks good!

Thank you! 🎉

**Pull request merged**